src/makeInfinityAdder.js

```diff
@@ -4,7 +4,27 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  let currentSum = 0;
+
+  const ADDER = (num) => {
```

Comment:
name functions in camelCase

Reply:
<img width="444" alt="image" src="https://github.com/mate-academy/js_task-inifinite-sum-closures/assets/70220667/aaab18f5-07ab-422e-9ae5-a8a52acbb23f">
still not fixed, feel free to ask me about it in the chat if you have some problems